google/resource_bigquery_table.go

```diff
@@ -268,6 +285,17 @@ func resourceTable(d *schema.ResourceData, meta interface{}) (*bigquery.Table, e
 		table.TimePartitioning = expandTimePartitioning(v)
 	}
 
+	fields := d.Get("binding").(*schema.Set)
```

Comment:
Ah, copy-paste error. I _think_ we need to use `Get` because `field` can be used multiple times – I copied this code from another place that had that behavior.